Admin node-sass version from 4.13.1 to 4.14.1

granitibrahimi opened the pull request:
Update packages.json due to errors encountered on Ubuntu: "Error: Node Sass does not yet support your current environment: Linux 64-bit with Unsuported Runtime (83)"

Updating the node-sass fixed the issue.


<!--
Thank you for contributing to Shopware! Please fill out this description template to help us to process your pull request.

Please make sure to fulfil our contribution guideline (https://docs.shopware.com/en/shopware-platform-dev-en/community/contribution-guideline?category=shopware-platform-dev-en/community).

Do your changes need to be mentioned in the documentation?
Add notes on your change right now in the documentation files in /src/Docs/Resources and add them to the pull request as well. 
-->

### 1. Why is this change necessary?
 Because it is failing to build administration via bin/build-administration.sh

### 2. What does this change do, exactly?
Updates npm package node-sass version from 4.13.1 to 4.14.1

### 3. Describe each step to reproduce the issue or behaviour.
Ubuntu 18.04 x86_64 GNU/Linux
run bin/build-administration.sh and you will get following error:
![image](https://user-images.githubusercontent.com/16402164/85272443-77f96980-b47c-11ea-996a-378e301d683a.png)

### 4. Please link to the relevant issues (if any).


### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

klarstil commented:
Thanks for your contribution @wolv-dev I added sorted list to your commit, so we're having the same behavior across both list types. The commit should be pushed in a few.

granitibrahimi committed:
Admin node-sass version from 4.13.1 to 4.14.1

Update packages.json due to errors encountered on Ubuntu: "Error: Node Sass does not yet support your current environment: Linux 64-bit with Unsuported Runtime (83)"

Updating the node-sass fixed the issue.

granitibrahimi committed:
Admin node-sass version from 4.13.1 to 4.14.1

Update packages.json due to errors encountered on Ubuntu: "Error: Node Sass does not yet support your current environment: Linux 64-bit with Unsuported Runtime (83)"

Updating the node-sass fixed the issue.

granitibrahimi committed:
Update package-lock.json

raknison commented on src/Administration/Resources/app/administration/package-lock.json:
```diff
@@ -1,6 +1,6 @@
 {
     "name": "administration",
-    "version": "1.0.0",
+    "version": "6.2.1",
```
@granitibrahimi Please can you revert this change? - The version should be `1.0.0`

raknison reviewed (changes requested):
Hi @granitibrahimi,

thank you for your contribution. Please revert the change of `version` in the `package-lock.json` file.

granitibrahimi committed:
Update package-lock.json

granitibrahimi commented:
> Hi @granitibrahimi,
> 
> thank you for your contribution. Please revert the change of `version` in the `package-lock.json` file.

Hi @raknison,

version is reverted to 1.0.0.

raknison commented:
> > Hi @granitibrahimi,
> > thank you for your contribution. Please revert the change of `version` in the `package-lock.json` file.
> 
> Hi @raknison,
> 
> version is reverted to 1.0.0.

@granitibrahimi Sorry, please squash your commits.

granitibrahimi committed:
Merge branch 'patch-1' of https://github.com/granitibrahimi/platform into patch-1

granitibrahimi commented:
> > > Hi @granitibrahimi,
> > > thank you for your contribution. Please revert the change of `version` in the `package-lock.json` file.
> > 
> > 
> > Hi @raknison,
> > version is reverted to 1.0.0.
> 
> @granitibrahimi Sorry, please squash your commits.

@raknison I tried at least lol, but I`m not sure I did it right. Probably I need a hand :(

raknison commented:
> > > > Hi @granitibrahimi,
> > > > thank you for your contribution. Please revert the change of `version` in the `package-lock.json` file.
> > > 
> > > 
> > > Hi @raknison,
> > > version is reverted to 1.0.0.
> > 
> > 
> > @granitibrahimi Sorry, please squash your commits.
> 
> @raknison I tried at least lol, but I`m not sure I did it right. Probably I need a hand :(

@granitibrahimi  I try to fix that for you later that day, allright?

granitibrahimi commented:
> > > > > Hi @granitibrahimi,
> > > > > thank you for your contribution. Please revert the change of `version` in the `package-lock.json` file.
> > > > 
> > > > 
> > > > Hi @raknison,
> > > > version is reverted to 1.0.0.
> > > 
> > > 
> > > @granitibrahimi Sorry, please squash your commits.
> > 
> > 
> > @raknison I tried at least lol, but I`m not sure I did it right. Probably I need a hand :(
> 
> @granitibrahimi I try to fix that for you later that day, allright?

Hi @raknison 

thank you very much. Would like to get a hint how this is done if that is ok with you!

Thanks

raknison commented:
Hi @granitibrahimi 

thank you for your contribution 🎉
Your PR was merged to 6.2

Best regards
Markus
